app/models/Reports.scala

```diff
@@ -6,14 +6,15 @@ import play.api.libs.json._
 
 object Reports {
 
-  val lists = Map[String, Int](
-    "guardianTodayUk" -> 37,
-    "guardianTodayUs" -> 1493,
-    "guardianTodayAu" -> 1506
+  val emailLists = Map[String, Int](
+    "guardianTodayUk_all" -> 111,
+    "guardianTodayUk_new" -> 16216,
+    "guardianTodayUs" -> 16125,
+    "guardianTodayAu" -> 2014
```

Comment:
There are called `EmailSendDefinition`(s) in exact target. Maybe we should rename it...